### PR TITLE
[CELEBORN-847] Support use RESTful API to trigger worker exit and exitImmediately

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
As title

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture @RexXiong @pan3793

**Comment by AngersZhuuuu** [comment]
ping @pan3793 @waitinfuture @RexXiong

File: service/src/main/scala/org/apache/celeborn/server/common/http/HttpRequestHandler.scala [hunk 1]
```diff
@@ -90,8 +94,8 @@ class HttpRequestHandler(
         service.isShutdown
       case "/isRegistered" if service.serviceName == Service.WORKER =>
         service.isRegistered
-      case "/decommission" if service.serviceName == Service.WORKER =>
-        service.decommission
+      case "/exit" if service.serviceName == Service.WORKER =>
+        service.exit(parameters.getOrElse("TYPE", ""))
```
Comment (AngersZhuuuu) [inline service/src/main/scala/org/apache/celeborn/server/common/http/HttpRequestHandler.scala:98]: Necessary to add strict constraint?

**Comment by AngersZhuuuu** [comment]
ping @pan3793 @waitinfuture @RexXiong

File: service/src/main/scala/org/apache/celeborn/server/common/http/HttpRequestHandler.scala [hunk 2]
```diff
@@ -17,6 +17,9 @@
 
 package org.apache.celeborn.server.common.http
 
+import java.net.URL
+import java.util.Locale
```
Comment (waitinfuture) [inline service/src/main/scala/org/apache/celeborn/server/common/http/HttpRequestHandler.scala:21]: Seems these two imports are unnecessary

File: docs/monitoring.md [hunk 3]
```diff
@@ -313,4 +313,4 @@ API path listed as below:
 | /unavailablePeers          | List the unavailable peers of the worker, this always means the worker connect to the peer failed.       |
 | /isShutdown                | Show if the worker is during the process of shutdown.                                                    |
 | /isRegistered              | Show if the worker is registered to the master success.                                                  |
-| /decommission              | Trigger this worker to decommission from the cluster                                                     |
\ No newline at end of file
+| /exit                      | Trigger this worker to exit, if worker enable graceful shutdown, it will exit gracefully.                |
```
Comment (waitinfuture) [inline docs/monitoring.md:316]: Better to document on the parameters

**Review by waitinfuture — APPROVED** [review]
LGTM except nits.

**Comment by waitinfuture** [comment]
Shall we consider idempotent here?

**Pull request closed without merging**